src/main/java/org/gridsuite/modification/server/entities/equipment/modification/OperatingStatusModificationEntity.java

```diff
@@ -20,43 +20,43 @@
 @NoArgsConstructor
 @Getter
 @Entity
-@Table(name = "branchStatusModification")
-@PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "branchStatusModification_id_fk_constraint"))
-public class BranchStatusModificationEntity extends EquipmentModificationEntity {
+@Table(name = "operatingStatusModification")
+@PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "operatingStatusModification_id_fk_constraint"))
```

Comment:
you don't have foreignKey name change in your liquibase changeset could you check this ?